hudi-common/src/main/java/org/apache/hudi/common/config/HoodieMetadataConfig.java

```diff
@@ -312,6 +312,13 @@ public final class HoodieMetadataConfig extends HoodieConfig {
       .withDocumentation("Maximum size in bytes of a single log file. Larger log files can contain larger log blocks "
           + "thereby reducing the number of blocks to search for keys");
 
+  public static final ConfigProperty<Boolean> DISABLE_FILESYSTEM_BOOTSTRAP = ConfigProperty
+      .key(METADATA_PREFIX + ".filesystem.bootstrap.disabled")
+      .defaultValue(false)
+      .sinceVersion("0.14.0")
+      .withDocumentation("Disable bootstrapping metadata table from the file system when the table is first created. "
```

Comment:
should we consider explicitly calling external metadata sync in the config naming. 
current naming "hoodie.metadata.filesystem.bootstrap.disabled" is kind of abstract. I am sure OSS dev may not understand what this config serves.

Reply:
also, should we prefix it with "_" for now so that we keep it internally. and so we avoid exposing to general users ?

Reply:
Any more thoughts here? I'm not sure about naming conventions

Reply:
We can mark this advanced. (@yihua to confirm) . 

On naming : just `hoodie.metadata.bootstrap.disabled` or `hoodie.metadata.auto.initialize` = true (default)| false?
Thats the best I can think of.

Reply:
hoodie.metadata.auto.initialize sounds better

Reply:
Going with `hoodie.metadata.auto.initialize`